### PR TITLE
fix: receiver address fix

### PR DESCRIPTION
**Pull request by MayukhSamadder2000**
fixed the receiver address issue

### CODE REVIEW

File: src/utils/transaction.ts [hunk 1]
```diff
@@ -423,6 +423,8 @@ export const honeyswapLP = async (
 ) => {
   if (!sdk) return { errorMessage: 'No sdk found' };
 
+  console.log('receiverAddress', receiverAddress);
```
Comment (IAmKio) [inline src/utils/transaction.ts:426]: Was this supposed to be here?

File: src/components/TransactionBlock/HoneySwapLPTransactionBlock.tsx [hunk 2]
```diff
@@ -185,7 +185,8 @@ const HoneySwapLPTransactionBlock = ({ id: transactionBlockId, errorMessages, va
       fromAssetIconUrl: selectedFromAsset?.logoURI,
       amount,
       accountType: selectedAccountType,
-      receiverAddress: receiverAddress ?? undefined,
+      receiverAddress:
+        (selectedReceiveAccountType === AccountTypes.Key ? providerAddress : accountAddress) ?? undefined,
```
Comment (IAmKio) [inline src/components/TransactionBlock/HoneySwapLPTransactionBlock.tsx:189]: What's the UI result of this being undefined here?

**Review by IAmKio — CHANGES_REQUESTED** [review]
Just a few small points

File: src/components/TransactionBlock/HoneySwapLPTransactionBlock.tsx (hunk 2 above)
Comment (MayukhSamadder2000) [reply src/components/TransactionBlock/HoneySwapLPTransactionBlock.tsx:189]: Nothing it won't let it pass through then during review

**Pull request merged**